examples/simple/src/index.js

```diff
@@ -29,6 +40,7 @@ render(
             />,
             <Route exact path="/custom2" component={CustomRouteLayout} />,
         ]}
+        customSagas={customSagas}
```

Comment:
seems overkill to use a saga just to display a form. Isn't there a more local solution?

Reply:
Same